framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -432,6 +467,28 @@ protected void addMainToArgs(final List<String> args) {
             args.add(quote(concatenatePaths(ppOpts)));
         }
 
+        if (PluginUtil.getJreVersion() > 8) {
+            // Get Patch_Module_argfile
+            final File patchModuleArgsFile = new File(jdkModulesPath, "Patch_Modules_argfile");
+
+            // Get currentJdkFolder
+            final String currentJdkFolder = jdkModulesPath.getPath();
+
+            // adding "--patch-module" arguments to compiler arguments
+            try {
```

Comment:
Can you change the `try` block to cover the smallest possible region? Isn't `PluginUtil.readFile` the only call that can cause an exception? Limiting the scope will make the error situations easier to understand.

Reply:
Done.